### ISSUE TITLE
attandance system

### ISSUE THREAD

**Issue by mjks4545** [mjks4545/seedsschool#3]
as explained

**Comment by mjks4545** [comment]
let me take over this issue i am working on this

**Comment by sherKhan1** [comment]
work that need to be done before attendance.
1):  Make Credentials to the admin to add teacher and assign to subjects.
2):  Admin add the student and give a subjects.
3): The classes to every teacher need to be assign by the admin.
4): And display every assigned classes to teacher and in every class in class need to be the students of that concern class. For example if class is 2nd class so there should only show the 2nd class students.
5): And after that the attendance will be in that specific.

# 

=============== What I have don today  ======================

```
The first 4 are done tonight by me.
```

**Comment by sherKhan1** [comment]
The attendance has been done. but its showing a little error. when student  present then he should be show present if any time open attendance page. And i have tried on this more then 1 hour but still unable to make so i will take help from my seniors about this tomorrow. and will done that.

**Comment by sherKhan1** [comment]
After that i will prepare the full month attendance list.and which could be convert to PDF also. But for that create new issue

**Comment by sherKhan1** [comment]
This is done... Completed
Code will share tomorrow
